recipes/vaex-hdf5/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "vaex-hdf5" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "d476d8825905efe8865ac4d747a1ec07f0be50c78e2c21bafd46565a127d5cd0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - vaex-core >=0.1
+    - h5py
+
+test:
+  imports:
+    - vaex.hdf5
+
+about:
+  home: https://www.github.com/maartenbreddels/vaex
```

Comment:
Where does the source for this actually live? Not seeing it in that repo....

Reply:
it's a branch (packages), and there it lives in the packages directory. If this goes onto conda-forge  and it's all working (and some other package coming) i'm merging it to master).

Reply:
Ah, cool. :+1: